.gitmodules

```diff
@@ -1,6 +1,12 @@
 [submodule "ocurrent"]
 	path = ocurrent
-	url = https://github.com/ocurrent/ocurrent.git
+	url = https://github.com/avsm/ocurrent.git
 [submodule "opam-0install-solver"]
 	path = opam-0install-solver
 	url = https://github.com/talex5/opam-0install-solver
+[submodule "ocaml-dockerfile"]
+	path = ocaml-dockerfile
+	url = https://github.com/avsm/ocaml-dockerfile
+[submodule "ocaml-version"]
+	path = ocaml-version
+	url = https://github.com/ocurrent/ocaml-version
```

Comment:
This will be ocaml-version.2.6.1 as soon as https://github.com/ocaml/opam-repository/pull/16822 is merged